src/h3/api/_api.pyi

```diff
@@ -0,0 +1,82 @@
+from typing import Dict, Iterable, List, Optional, Set, Tuple, TypeVar
+
+from typing_extensions import Literal
+
+# Static Types
+AreaUnits = Literal['km^2', 'm^2', 'rads^2']
+LengthUnits = Literal['km', 'm', 'rads']
+Resolution = int
+GeoBoundary = Tuple[Tuple[float, float], ...]
+KRange = int
+
+# Type variables
+H3Cell = TypeVar('H3Cell')
+H3Edge = TypeVar('H3Edge')
+UnorderedH3Cell = TypeVar('UnorderedH3Cell')
+
+def versions() -> Dict[str, str]: ...
+def string_to_h3(h: str) -> int: ...
+def h3_to_string(x: int) -> str: ...
```

Comment:
Will this accept `H3Cell` as input?

Reply:
Just noting this was typed as `int` because `h3_to_string` always takes an `int`, no matter the API
https://github.com/uber/h3-py/blob/7364b21371386d800a550bdf370669782e31ffdb/src/h3/api/_api_template.py#L100-L101